### PR TITLE
updated the readme 

### PR DESCRIPTION
**Pull request by BriannaGreene**
to make the deployment process clearer for new employees

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,18 +21,29 @@ This is a collection of tasks executed with the Jake task runner. See http://jak
...
+#### Example Deployment Command
 Deploy the "console" app to the "staging-uat" ECS cluster.
+(Staging and Production cluster names can be found on AWS in ECS Clusters)
 ```bash
 jake deploy:app['staging-uat','console']
 ```
 
-### To Deploy Shipping to Staging UAT
+## How to Deploy the Shipping Platform to Staging
 1. create a staging branch
```
Comment (casthewiz) [inline README.md]: Any naming conventions exist for the branch we're supposed to create?

Comment (BriannaGreene) [reply README.md]: Usually, it's something like `staging/name-of-current-sprint`. The Stability team (JD, Tom, and David) will most likely be the only ones deploying the Shipping Platform, so no need to worry about that. I will add an example for clarity.

File: README.md [hunk 2]
```diff
@@ -21,18 +21,29 @@ This is a collection of tasks executed with the Jake task runner. See http://jak
...
+(Staging and Production cluster names can be found on AWS in ECS Clusters)
 ```bash
 jake deploy:app['staging-uat','console']
 ```
 
-### To Deploy Shipping to Staging UAT
-1. create a staging branch
+## How to Deploy the Shipping Platform to Staging
+1. create a staging branch (Example: `staging/name-of-current-sprint`)
 2. merge in branch `GCT-314-get-shipping-working` (this branch is not in prod)
```
Comment (siennamw) [inline README.md]: Is this step still needed?  I don't see a branch with that name in the shipping repo anymore.

Comment (BriannaGreene) [reply README.md]: Yes, I checked with them.

**Pull request merged**